### PR TITLE
NEXUS-25050 Helm format. Administrative

### PR DESCRIPTION
**Pull request by mlukaretkyi**
https://issues.sonatype.org/browse/NEXUS-25050

### CODE REVIEW

**Review by j-s-3 — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -22,6 +22,14 @@
 
 [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-helm/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** Helm is now part of Nexus Repository Manager. Version 3.21 includes the Helm plugin by default. 
+>The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-helm](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-helm).
+
+> **Filing issues:** Upgrade to the latest version of Nexus Repository Manager 3, to get the latest fixes and improvements, before filing any issues or feature requests at https://issues.sonatype.org/.
+
+> **Upgrading:** If you are using a version prior to 3.21 and upgrade to a newer version you will not be able to install the community plugin. 
+>No other changes are required, and your existing data will remain intact.
```
Comment (joedragons) [inline README.md:31]:
```suggestion
>No other changes are required and your existing data will remain intact.
```
Rationale: This is not an Oxford comma situation, it is just 2 items, so IMO (trivial) opinion, just a redundant conjunction.

**Review by joedragons — APPROVED** [review]
+1 w/ assumption https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-helm will eventually work (currently 404).

File: README.md (hunk 1 above)
Comment (mlukaretkyi) [reply README.md]: Fixed. https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/167/commits/525dd9981fb51ef0fb1d769fd51cbdf1325fc4c2

**Pull request merged**